src/plugins/auto-closer/index.js

```diff
@@ -0,0 +1,172 @@
+ * @returns {Promise<boolean>} `true` if the repository has a label with appropriate name
+ */
+async function hasAutoCloseLabel(context) {
+    const allLabels = await context.github.paginate(
+        context.github.issues.getLabels(context.repo()),
+        res => res.data
+    );
+
+    return allLabels.some(label => label.name === AUTO_CLOSE_LABEL);
+}
+
+
+/**
+ * Creates a search query to look for issues in a given repo that have been
+ * labeled "accepted", don't have an assignee/project/milestone, have been opened
+ * for 90 days and have been inactive for 30 days.
+ * @param {string} options.owner The owner of the repo where issues should be searched
+ * @param {string} options.repo The name of the repo where issues should be searched
+ * @returns {string} A search query to send to the GitHub API
+ */
+function createAutoCloseAcceptedSearchQuery({ owner, repo }) {
+    const creationCutoffDate = moment().subtract({ days: ACCEPTED_AUTO_CLOSE_DAYS });
+    const activityCutoffDate = moment().subtract({ days: ACTIVITY_AUTO_CLOSE_DAYS });
+
+    return [
+        "is:open",
+        `repo:${owner}/${repo}`,
+        "no:assignee no:milestone no:project label:accepted",
+        `created:<${creationCutoffDate.format("YYYY-MM-DD")}`,
+        `updated:<${activityCutoffDate.format("YYYY-MM-DD")}`
+    ].join(" ");
+}
+
+/**
+ * Creates a search query to look for issues in a given repo that have not been
+ * labeled "accepted", don't have an assignee/project/milestone, have been
+ * inactive for 30 days.
+ * @param {string} options.owner The owner of the repo where issues should be searched
+ * @param {string} options.repo The name of the repo where issues should be searched
+ * @returns {string} A search query to send to the GitHub API
+ */
+function createAutoCloseUnacceptedSearchQuery({ owner, repo }) {
+    const activityCutoffDate = moment().subtract({ days: ACTIVITY_AUTO_CLOSE_DAYS });
+
+    return [
+        "is:open",
+        `repo:${owner}/${repo}`,
+        "no:assignee no:milestone no:project -label:accepted",
+        `updated:<${activityCutoffDate.format("YYYY-MM-DD")}`
+    ].join(" ");
+}
+
+/**
+ * Creates the bot comment to leave on auto-closed accepted issues.
+ * @returns {string} comment message
+ * @private
+ */
+function createAcceptedAutoCloseMessage() {
+    return `
+Unfortunately, it looks like there wasn't enough interest from the team
+or community to implement this change. While we wish we'd be able to
+accommodate everyone's requests, we do need to prioritize. We've found
+that accepted issues failing to be implemented after 90 days tend to
+never be implemented, and as such, we [close those issues](https://eslint.org/docs/maintainer-guide/issues#when-to-close-an-issue).
+This doesn't mean the idea isn't interesting or useful, just that it's
+not something the team can commit to.
+
+Thanks for contributing to ESLint and we appreciate your understanding.
+
+[//]: # (auto-close)
```

Comment:
What does this line do, out of curiosity?

Reply:
I have no idea! I just noticed that the `auto-archiver` had that in its comment, so I assumed it was used for something and left it in. :)

Reply:
This is a unique hash used by `duplicate-comments` plugin to remove older duplicate comments.